<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="btnPlay">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/gfx/icons/btnGo.png</normaloff>:/gfx/icons/btnGo.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnPause">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/gfx/icons/btnPause.png</normaloff>:/gfx/icons/btnPause.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnStop">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/gfx/icons/btnStop.png</normaloff>:/gfx/icons/btnStop.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnAdd">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/gfx/icons/btnOpen.png</normaloff>:/gfx/icons/btnOpen.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnPrev">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/gfx/icons/btnPrevPlaylistItem.png</normaloff>:/gfx/icons/btnPrevPlaylistItem.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnNext">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/gfx/icons/btnNextPlayListItem.png</normaloff>:/gfx/icons/btnNextPlayListItem.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnSeekBackwards">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/gfx/icons/btnSlower.png</normaloff>:/gfx/icons/btnSlower.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnSeekForwards">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/gfx/icons/btnFaster.png</normaloff>:/gfx/icons/btnFaster.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblVolume">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Volume:50%</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="volumeSlider">
         <property name="minimumSize">
          <size>
           <width>81</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>81</width>
           <height>16</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxPlaybackMode"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lblCurrentTime">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblRemainingTime">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSlider" name="seekSlider">
       <property name="minimumSize">
        <size>
         <width>341</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="minimumSize">
        <size>
         <width>341</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tableViewPlaylist"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Current track:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblCurrentTrack">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnLoadPlaylist">
         <property name="text">
          <string>Load playlist</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSavePlaylist">
         <property name="text">
          <string>Save playlist</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRemoveSelected">
         <property name="text">
          <string>Remove selected</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnClearPlaylist">
         <property name="text">
          <string>Clear playlist</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnQuit">
         <property name="text">
          <string>Quit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="action_remove_selected_tunes_from_playlist">
   <property name="text">
    <string>Remove selected tunes from playlist</string>
   </property>
  </action>
  <action name="actionClear_playlist">
   <property name="text">
    <string>Clear playlist</string>
   </property>
  </action>
  <action name="action_remove_selected_files_from_HDD">
   <property name="text">
    <string>remove selected files from HDD</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnQuit</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>413</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>551</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
